src/annotations/components/controlBar/AnnotationsControlBar.tsx

```diff
@@ -29,25 +27,23 @@ export const AnnotationsControlBar: FC = () => {
   // make both textblocks have 'inline' style to get them on the same line; else there is a line break.
   // using two elements to put space between them.
   return (
-    <FlexBox
+    <Toolbar
       testID="annotations-control-bar"
-      justifyContent={JustifyContent.FlexStart}
       margin={ComponentSize.Large}
+      style={{justifyContent: 'center', padding: '0 32px', width: '100%'}}
```

Comment:
how'd you come up with the padding here? are there clockface defaults we can use?

@daviddkkim

Reply:
i put in something that looks good.  @daviddkkim  will adjust after.